### DISCUSSION TITLE
Unable to purchase license

### DISCUSSION THREAD

**Discussion by zenjubbers** [MobiusSync/MobiusSync#94] [Q&A] [answered]
When trying to buy  the license to remove the 20MB file size restriction, I get a popup on my iPad that appears garbled and nothing is clickable. I am not sure how to purchase the license to remove the file size restriction on free.

![image](https://user-images.githubusercontent.com/66967532/216418277-20bd4b15-2d7f-4dd8-ae8a-b94ae91054bb.png)

I should be on the very latest version of the app as it was just installed from the app store last night.

Let me know if there is another way to purchase the license.

**Comment by cmmartti** [comment]
Have you increased the font size in the iOS system settings? Some application developers never test for this and and as a result they create inflexible layouts that break if the font size is increased.

If you add the font size widget to the pull-up quick settings panel, you can change the font size for each app individually.

**Comment by zenjubbers** [comment]
It's my wife's iPad. Very doubtful she even knows the font size can be adjusted but I'll check. Even then the text might not be readable but clicking to purchase would still work no?

**Comment by zenjubbers** [comment, marked as answer]
Ahh. Figured it out. Wasn't font but text size. Made it smaller and the purchase pop up displays correctly with a functional link now. Thanks for the hint.
